.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+# Pull Request
+
+## Summary
+
+Provide a concise and clear summary of your pull request.
+
+## Motivation
+
+Explain the motivation behind this pull request. What problem does it solve or feature does it introduce?
+
+## Changes Made
+
+Describe the changes made in this pull request. Be specific about code changes, new features, or bug fixes.
+
+## Related Issues/PRs
+
+Link to any related issues or pull requests, if applicable.
+
+## Screenshots (Before/After if applicable)
+
+Include screenshots or visual aids to demonstrate the changes visually. If your PR modifies the UI or behavior, include before and after descriptions or screenshots.
+
+### Desktop
+
+### Mobile
+
+### iPad
+
+## Checklist
+
+- [x] I have read the [Contributing Guidelines](../../CONTRIBUTING.md) and followed them.
+- [ ] I have tested my changes thoroughly.
+- [ ] My code follows the project's coding standards and style guide.
+- [ ] I have updated documentation (if applicable).
```

Comment:
```suggestion
- [x] I have read the [Contributing Guidelines](../../CONTRIBUTING.md) and followed them.
- [ ] Unit tests were added.
```

- I removed the "coding standards" part because it's already covered by eslint and prettier on CI (automated check)
- I removed the documentation because currently, that lives outside the repo as explained in the Contributing file.

Reply:
I have made the suggested changes.